src/traces/heatmap/hover.js

```diff
@@ -10,12 +10,13 @@
 'use strict';
 
 var Fx = require('../../plots/cartesian/graph_interact');
+var constants = require('../../plots/cartesian/constants');
 var Lib = require('../../lib');
 
 
 module.exports = function hoverPoints(pointData, xval, yval, hovermode, contour) {
     // never let a heatmap override another type as closest point
-    if(pointData.distance < Fx.MAXDIST) return;
+    if(pointData.distance < constants.MAXDIST) return;
```

Comment:
@etpinard I didn't notice you fixed this issue in #655 too - but when I merged master back in I used your version, which is cleaner.